dts/bindings/dma/st,stm32-dma-v3.yaml

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32 DMA controller (V3)
+
+  It is present on stm32 devices stm32U5
+  This DMA controller includes several channels with different requests.
+  DMA clients connected to the STM32 DMA controller must use the format
+  described in the dma.txt file, using a four-cell specifier for each
```

Comment:
dma.txt file doesn't exist.

Reply:
line removed

Reply:
Same desrciption exists for all the st,stm32-dma-v_X_.yaml